docs/articles/documentation/test-api/test-code-structure.md

```diff
@@ -277,6 +281,54 @@ fixture `MyFixture`
     .page `../my-project/index.html`;
 ```
 
+## Specifying Testing Metadata
+
+TestCafe allows you to specify additional information for fixtures and tests in the form of *field-value metadata*.
+
+### Fixture Metadata
+
+You can specify metadata for a fixture by using the `meta` method in the [fixture declaration](#fixtures).
+
+```js
+fixture.meta({ field1: 'value1', field2: 'value2', field3: 'value3' });
+```
+
+Parameter  | Type   | Description
+---------- | ------ | -----------------
+`metadata` | Object | Field-value pairs
+
+**Example**
+
+```js
+fixture 'MyFixture'
+    .meta('fixtureID', 'f-0001')
+    .meta({ author: 'John', creationDate: '05/03/2018' });
+```
+
+### Test Metadata
+
+You can specify metadata for an individual test by using the `test.meta` method.
```

Comment:
We also should note, that `.meta()` calls override values for entries specified in previous `.meta()` calls and it doesn't matter if it was `fixture.meta()` or `test.meta()` call. All metadata entries specified for a fixture are available in tests that reside in the fixture.